frontend/src/components/staff/RequestTable.jsx

```diff
@@ -56,8 +57,24 @@ export default function RequestTable({ type, refresh, onRefresh }) {
     });
   }
 
+  if (type === 'unfulfilled') {
```

Comment:
nit: a switch statement is more appropriate here (the functionality is unchanged, but code may be more readable)